README.md

```diff
@@ -1 +1,32 @@
-# INF 6420 - Intro to Web Development
+# Courteney Shovlin's INF6420 Project
+
+This site will be an introduction to myself and a showcase of what I have done in my life, thusfar. This will include my academic, professional, and personal life highlights.
+
+## Wireframes
+
+Planning will be an instrumental part to the success of this project. Below I have included some wireframes to serve as an example of the possible organization/structure of my pages.
+
+![Wireframe of Landing/Home Page](wireframes\Wireframe 1.jpg)
```

Comment:
Hi @courteshov the images are broken. If you change `\` with `/`, that will resolve the issue

Reply:
Hello! I apologize for my delay. I have been traveling. I believe I have it fixed now.